src/BlazorUI/Bit.BlazorUI/Components/Navs/Nav/BitNav.scss

```
@@ -3,6 +3,12 @@
 .bit-nav {
     color: $clr-fg-pri;
     font-family: $tg-font-family;
+    // The size tokens live on the root so a single class swaps the metrics of every part at once.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)